packages/react/spec/i18n.spec.js

```diff
@@ -2,30 +2,38 @@
 import i18n from '../src/javascripts/lib/i18n'
 
 const mockEN = {
-  'one': 'the first translation',
+  one: 'the first translation',
```

Comment:
Did this trigger a lint warning or something?

Reply:
oh.. no, I should revert it back in both PR :D nice pick, it does not change anything, just me for testing something before

Reply:
Addressed  :)

Reply:
Oh, it is lint fix actually. I will ignore this change then :)